.circleci/config.yml

```diff
@@ -793,31 +789,54 @@ workflows:
           requires:
             - playwright-build
 
-  build-and-deploy-all-apps-workflow:
-    when:
-      and:
-        - not: << pipeline.parameters.do-builds >>
-        - not: << pipeline.parameters.scheduled-playwright >>
+  build-test-deploy-apps:
+    # conditionally build and deploy for changed studies on develop branch
     jobs:
-      - conditionally-launch-build-and-deploy-all-job:
+      - app-build:
           <<: *filter-develop-branch
-
-  build-and-deploy-workflow:
```

Comment:
`build-and-deploy-workflow` workflow is no longer used. See comment above. It was triggerd by CI api call from inside the`build-and-deploy-all-apps-workflow` workflow job.